cockatrice/src/client/tabs/tab_deck_editor.cpp

```diff
@@ -852,6 +864,20 @@ void TabDeckEditor::updateHash()
     hashLabel->setText(deckModel->getDeckList()->getDeckHash());
 }
 
+void TabDeckEditor::updateRecentlyOpened()
+{
+    openRecentsMenu->clear();
```

Comment:
Does it make sense to try and deleteLater/Disconnect the items in the list before we clear it out and add new pointers? I'm not sure if Qt is handling this destructions or not. Might need to valgrind look.